src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx

```diff
@@ -59,37 +61,31 @@ export default function AppealListItemV3({ appeal, name, external = false }) {
   // "Appeal updated on March 6, 2019"
   // "Disability Compensation Appeal"
 
-  let appealTitle = '';
+  let appealTitle = getTypeName(appeal);
   let updatedOn = '';
 
-  if (isAppeal) {
-    if (programArea) {
-      appealTitle = `${programArea} `;
-    }
-    appealTitle += getTypeName(appeal);
-  } else {
-    appealTitle = getTypeName(appeal);
-    if (programArea) {
+  if (programArea) {
+    if (isAppeal) {
```

Comment:
I'm wondering why `isAppeal` is `false` for HLR & Supplemental Claim, and true for NOD & NOD (legacy type). We work on NOD, HLR & SC and we call all three an appeal (or decision review).

Reply:
That's a good question, honestly would be worth a look at by the content team maybe?